wkcuber/metadata.py

```diff
@@ -12,6 +12,8 @@
 from .mag import Mag
 from typing import List
 from .utils import add_verbose_flag, setup_logging, add_scale_flag
+from pathlib import Path
+from os.path import *
```

Comment:
I consider `import *` as an anti-pattern as it hides the fact where functions are taken from. For example, the code itself doesn't really tell me that `glob` stems from `os.path`. Additionally, there is the risk that this wildcard import shadows other variables in this namespace. Therefore, I'd advise to state explictly what you want to import. Alternatively, you can just do `import os.path` and then use glob like `os.path.glob`, I think.

Reply:
I see what you mean. Actually glob is not from `os.path` (it was already imported).

Reply:
> Actually glob is not from `os.path` (it was already imported).

that drives the point home, too :laughing:

Reply:
#advertisment Me likey `pathlib.rglob` https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob

<img src="https://media.giphy.com/media/d2YZzTQvyoNYf9YI/200w_d.gif">